cfgov/v1/models/home_page.py

```diff
@@ -289,8 +68,21 @@ class HomePage(CFGOVPage):
     # Tab handler interface
     edit_handler = TabbedInterface([
         ObjectList(content_panels, heading='General Content'),
-        ObjectList(CFGOVPage.sidefoot_panels, heading='Sidebar'),
         ObjectList(CFGOVPage.settings_panels, heading='Configuration'),
+        ObjectList([
+            InlinePanel(
+                'carousel_items', min_num=4, max_num=4, label="Carousel Item"
+            ),
+        ], heading='Carousel'),
+        ObjectList([
+            InlinePanel(
+                'info_units', min_num=6, max_num=6, label="Info Unit"
+            ),
+        ], heading='Info Units'),
+        ObjectList([
+            FieldPanel('card_heading'),
+            InlinePanel('cards', min_num=3, max_num=3, label="Card"),
+        ], heading='Cards')
```

Comment:
Suggest putting these before Configuration to keep the Configuration tab consistently on the far right for all page types.